core/common/textutil.hpp

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_CORE_COMMON_TEXTUTIL_HPP
+#define CPP_FILECOIN_CORE_COMMON_TEXTUTIL_HPP
+
+#include <string>
+#include <type_traits>
+
+namespace textutil {
+  /**
+   * @brief Convert boost::multiprecision big integers to string
+   * @param number input data
+   * @return output string
+   */
+  template <class T,
+            std::enable_if_t<(sizeof(T) > sizeof(int64_t))> * = nullptr>
```

Comment:
Does it work correctly with signed numbers? Should it work with signed numbers?

Reply:
LEB128 codec was implemented only for unsigned values, algorithm for signed values is different